.travis.yml

```diff
@@ -21,20 +21,19 @@ cache:
 env:
   matrix:
     # These apply to the latest WP version that is available as a Docker image: https://hub.docker.com/_/wordpress/
-    - PHP_VERSION=5.6
```

Comment:
🤔 

So since we support PHP 5.6+, I want to make sure we know if we have tests breaking for 5.6. 

I'm not 💯 opposed to bumping our minimum version, but with WordPress core moving to PHP 5.6 min version in April, it seems to make sense to continue to support 5.6+ unless we have a serious need to change.

Reply:
Ok I'll use WP 5.0.2. That version still has PHP 5.6-based Docker images.